test/smoke/alpha.test.ts

```diff
@@ -0,0 +1,124 @@
+            request.mode
+        ]
+    );
+
+    const appointmentBytes = ethers.utils.defaultAbiCoder.encode(
+        ["bytes", "bytes", "bytes"],
+        [basicBytes, callBytes, conditionBytes]
+    );
+
+    return ethers.utils.keccak256(appointmentBytes);
+};
+
+describe("alpha", () => {
+    const PISA_URL = "http://18.219.31.158:5487/appointment";
+    const ROPSTEN_URL = "https://ropsten.infura.io/v3/e587e78efcdd4c1eb5b068ee99a6ec0b";
+
+    const createAppointmentRequest = (
+        contractAddress: string,
+        customerAddress: string,
+        data: string,
+        eventAbi: string,
+        eventArgs: string,
+        id: number,
+        jobId: number,
+        startBlock: number
+    ) => {
+        return {
+            challengePeriod: 100,
+            contractAddress,
+            customerAddress: customerAddress,
+            data,
+            endBlock: startBlock + 30,
+            eventABI: eventAbi,
+            eventArgs: eventArgs,
+            gasLimit: "100000",
+            id,
+            jobId,
+            mode: 1,
+            preCondition: "0x",
+            postCondition: "0x",
+            refund: "0",
+            startBlock,
+            paymentHash: "0xfc1624bdc50da30f2ea37b7debabeac1f6166db013c5880dcf63907b04199138"
+        };
+    };
+
+    it("pisa", async () => {
+        // connect to ropsten
+        const provider = new JsonRpcProvider(ROPSTEN_URL);
+
+        // 0xC73e1ebaFE312F149272ccA46A4acA3F8e8C62A6
+        const customer = new Wallet("0xD3E0200D9A8E615ED48E8317730EDD239BCDE54FB6EB2EBDC2FD6E6EA57AD6B3", provider);
+
+        // deploy the contract
+        const channelContractFactory = new ethers.ContractFactory(ABI, ByteCode, customer);
+        const rescueContract = channelContractFactory.attach("0x717Bd700367AEBf70a6e37ca731937c8079D0047");
+        // const rescueContract = await channelContractFactory.deploy();
+
+        // setup
+        const startBlock = await provider.getBlockNumber();
+        const message = "sos";
+        const id = 8;
+        const jobId = 1;
+
+        // create an appointment
+        const appointmentRequest = createAppointmentRequest(
+            rescueContract.address,
+            customer.address,
+            encodeData("remote"),
+            DISTRESS_EVENT_ABI,
+            encodeArgs(message),
+            id,
+            jobId,
+            startBlock
+        );
+
+        // encode the request and sign it
+        const encoded = encode(appointmentRequest);
+        const customerSig = await customer.signMessage(ethers.utils.arrayify(encoded));
+
+        const response = await request.post(PISA_URL, {
+            json: { ...appointmentRequest, customerSig }
+        });
+        console.log(response)
+        
+        const tx = await rescueContract.help()
+        console.log("help broadcast")
+        await tx.wait();
+        console.log("help mined")
+        return;
+    }).timeout(100000);
+});
```

Comment:
Is this test incomplete? Doesn't seem to be waiting for Pisa's response.

Reply:
Yep this was missing the check! This is just a smoke test to run against ropsten - updated now, but still wont work at the moment as I still need to deploy the pisa contract on ropsten.